src/services/adalService.js

```diff
@@ -26,12 +26,15 @@ export const getAuthContext = (dispatch) => {
 export const login = (dispatch) => {
   if (typeof window !== 'undefined' && !!window) {
     getAuthContext(dispatch).login()
+
+    appInsights.trackEvent("SIA Login")
```

Comment:
These should be tracked in authNService rather than adalService, so we don't have to add tracking in each of adalService, msalService, and any other authentication service options we want to add. The statements here can be moved to the login and logOut methods in authNService before the switch statements to record events for all authentication types.